app/Repositories/BuildRepository.php

```diff
@@ -165,7 +165,19 @@ public function getByPlatform(Application $application): Collection
             ->latest()
             ->get()
             ->groupBy('platform')
-            ->sortKeys();
+            ->sortKeys()
+            ->map(function (Collection $platformCollection) {
+                return $platformCollection->map(function (Build $build) {
+                    if (! $build->partial_rollout && $build->rollout_percentage !== 100) {
+                        $build->rollout_percentage = 100;
+                    }
+                    $build->installations_percent = $build->downloads > 0 ?
+                        (int) floor(($build->installations / $build->downloads) * 100) :
```

Comment:
Here I'd like to change the denominator, like this:

```suggestion
(int) floor(($build->installations / nr_of_people_impacted_by_rollout) * 100) :
```

For `nr_of_people_impacted_by_rollout` I mean the percentage of "active devices".

For reference: https://github.com/everli/uppy/blob/12d92564605fc71c3c5483a66ba3655dbabf0d12/app/Repositories/BuildRepository.php#L253 ). 

For example: if the rollout is 30% -> then 30% of active devices.

I hope I explained it well 🙏

Reply:
I think I got